services/frontend-service/pnpm-lock.yaml

```diff
@@ -11530,17 +11539,17 @@ packages:
       open: 7.4.2
       pkg-up: 3.1.0
       prompts: 2.4.0
-      react-error-overlay: 6.0.11
+      react-error-overlay: 6.0.9
```

Comment:
Why the downgrade?

Reply:
The `error-overlay` pacakge was updated to support webpack v5, and is now incompatible with react-scripts v4 which we still need.

Reply:
Fine for now, but next time, please put such a comment into the code and git commit message